literature.bib

```diff
@@ -4096,7 +4096,7 @@ @inproceedings{LG:VaMoS15
 @inproceedings{M:SE15,
 	author = {Mustafa Al-Hajjaji},
 	title = {{Scalable Sampling and Prioritization for Product-Line Testing}},
-	booktitle =  {Doktorandensymposium der Software Engineering},
+	booktitle = {Doktorandensymposium der Software Engineering},
```

Comment:
same here. Is this the complete name?

Reply:
I don't know, I just removed the additional space.

Reply:
https://github.com/tthuem/BibTags/blame/master/literature.bib#L3985